### ISSUE TITLE
Fatal Error Uncaught Error: Call to a member function get() on null 

### ISSUE THREAD

**Issue by miklb** [snarfed/wordpress-micropub#139]
in branch master 5737f6e62f9d901748e341d8e38475ec51d9ed21

```
[18-Jul-2018 02:37:00 UTC] PHP Fatal error:  Uncaught Error: Call to a member function get() on null in /wp-includes/query.php:28
 
Stack trace:
 
#0 plugins/micropub/includes/class-micropub-authorize.php(124): get_query_var('micropub')
 
#1 wp-includes/class-wp-hook.php(286): Micropub_Authorize::determine_current_user(false)
 
#2 wp-includes/plugin.php(203): WP_Hook->apply_filters(false, Array)
 
#3 /wp-includes/user.php(2603): apply_filters('determine_curre...', false)
 
#4 /wp-includes/pluggable.php(69): _wp_get_current_user()
 
#5 /wp-includes/user.php(433): wp_get_current_user()
 
#6 /wp-includes/class-wp-query.php(2272): get_current_user_id()
 
#7 /wp-includes/class-wp-query.php(3230): WP_Query->get_posts()
 
#8 /wp-includes/post.php(1 in //wp-includes/query.php on line 28
```
running along master of IndieAuth plugin

**Comment by dshanske** [comment]
I cannot reproduce this and I'm running the same masters. Also, if Indieauth is active, ti should never load that file at all, and I verified that is the case. Will continue to try to figure this out though.

**Comment by dshanske** [comment]
Resolved via #162
